process/factory/shard/vmContainerFactory_test.go

```diff
@@ -70,9 +69,9 @@ func TestVmContainerFactory_Create(t *testing.T) {
 	assert.Nil(t, err)
 	assert.NotNil(t, container)
 
-	vm, err := container.Get(factory.IELEVirtualMachine)
+	/*vm, err := container.Get(factory.IELEVirtualMachine)
```

Comment:
Add a TODO to fix IELE. sometimes.

Reply:
it was fixed. has to put back into the test as well.